spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Omar') }
+  before { subject.save }
+
+  it 'Name should be present' do
+    subject.name = nil
+    expect(subject).to_not(be_valid)
+  end
+
+  it "shouldn't have any recent posts" do
+    expect(subject.recent_posts.length).to eq(0)
+  end
+
+  it 'should have a positive posts counter' do
```

Comment:
- [ ] @sagieramos Great work implementing the changes requested; as the previous reviewer requested kindly add a test that shows the `recent_posts` method is displaying three recent posts when called, this will require creating four posts and checking if it's displaying the 3 recent posts.